frontend/src/react/App/App.css

```diff
@@ -1,13 +1,33 @@
 body {
   background: #ECF0F3;
 }
-
+.is-white-important {
+  background: white !important;
+}
 .is-a-link {
-  color: #3273dc;
+  color: #3273dc !important;
```

Comment:
Ils sont indispensables les `!important` ici ?
parce que c'est plutôt #badPractice de faire ça :(

Reply:
One is necessary, I've removed the other ones, thx